include/mls/credential.h

```diff
@@ -109,9 +109,7 @@ class Credential
 
 namespace tls {
 
-using namespace mls;
-
-TLS_VARIANT_MAP(CredentialType, BasicCredential, basic)
-TLS_VARIANT_MAP(CredentialType, X509Credential, x509)
+TLS_VARIANT_MAP(mls::CredentialType, mls::BasicCredential, basic)
+TLS_VARIANT_MAP(mls::CredentialType, mls::X509Credential, x509)
```

Comment:
Why are the `mls::` prefixes required here, but not in `messages.h`?

Reply:
compiler reports error on only this. other it was throwing error if i use mls::